xml/System.Numerics/BigInteger.xml

```diff
@@ -1482,11 +1482,15 @@
 ## Remarks  
  This property is a convenience feature that indicates whether a <xref:System.Numerics.BigInteger> value is evenly divisible by two. It is equivalent to the following expression:  
   
-```csharp  
+ [C#]  
```

Comment:
@Rowdel, can you access the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C24%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201705240709352752-2236%5Cworkflow_report.html)? It shows build errors in 16 files, all of them caused by failing to close the CDATA element. In each case, the `</format>` tag should be preceded by `]]>` to close the CDATA element.